python_modules/libraries/dagster-powerbi/dagster_powerbi/translator.py

```diff
@@ -78,6 +78,23 @@ class PowerBIContentData:
     properties: Dict[str, Any]
 
 
+class PowerBITranslatorData(NamedTuple):
```

Comment:
can just make this `@record` but not serializable to be consistent with the rest of this interface.

Reply:
Done in [7c63440](https://github.com/dagster-io/dagster/pull/26654/commits/7c634402dce3a24b24b92936f1ba763a68870262)